cmd/skupper/skupper.go

```diff
@@ -420,6 +421,18 @@ installation that can then be connected to other skupper installations`,
 				}
 			}
 
+			if routerCreateOpts.ImageRegistry != "" {
+				urlImageRegistry, err := url.Parse(routerCreateOpts.ImageRegistry)
+				if err != nil {
+					return fmt.Errorf("The specified image registry is not valid: %s", err)
```

Comment:
This validation does not work properly. If you have a registry running on a custom port,
you usually set it to, in example: `10.0.0.1:5000`. The url.Parse expects the protocol to be defined.

Maybe it can be added temporarily as a prefix but removed from the final value to be used.

Reply:
Thanks! I will try to parse the url using a regexp instead.